[FEATURE] Added notifier to backend

CATBIRDS opened the pull request:
Adds the Notifier python script to the backend.

CATBIRDS committed:
Create notifier.py

Initial commit for the Notifier.

yourbuddyconner commented on packages/fork-checker/backend/notifier/notifier.py:
```diff
@@ -0,0 +1,260 @@
+        forkdata = {"length": len(fork), "blocks": [], "creators": [], "rewards": 0, "latest": '', "last_updated": 0}
+        for block in fork:
+            forkdata["blocks"].append(block)
+            forkdata["creators"].append(blocks[block]["protocol_state"]["body"]["consensus_state"]["block_creator"])
```
Have this only append the creator if it is not already present in the list

CATBIRDS committed:
Updated Notifier + Requirements.txt

Requirements.txt:
- Added requirements.txt
Notifier.py:
- Removed unused imports
- Fork database information now only includes a block creator's hash once per fork
- Reformatted environmental variable usage to include reasonable defaults
- Replaced pyrebase wrapper with the official firebase_admin SDK
- Changed firebase authentication method to mirror API's process for convenience
- Added main scope for running as a standalone script

yourbuddyconner commented on packages/fork-checker/backend/notifier/requirements.txt:
```diff
@@ -0,0 +1 @@
+firebase_admin
```
Please add graph_tool and glob here as well

yourbuddyconner committed:
last_updated str to int

CATBIRDS committed:
Update notifier.py

Reverse order of blocks being sent to the database such that the oldest blocks are first.
